ddpui/models/resource_share.py

```
@@ -0,0 +1,78 @@
+        related_name="resource_shares_created",
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        db_table = "resource_share"
+        indexes = [
+            # "who has access to this resource?" — tenant-scoped resource lookup
+            models.Index(fields=["org", "resource_type", "resource_id"]),
+            # "what does this principal have access to?" — tenant-scoped principal lookup
+            models.Index(fields=["org", "principal_type", "principal_id"]),
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)